domains/fbi.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "2Much4U's Website",
+  "repo": "https://github.com/2m4u",
+  "owner": {
+    "username": "2M4U",
+    "email": "2m4u@fbi.is-a.dev"
+  },
+  "record": {
+    "CNAME": "2m4u.netlify.app"
```

Comment:
Netlify/Vercel will not work with a `CNAME` record, consider switching to [Railway](https://railway.app/?referralCode=jj), or use a `URL` record.

Reply:
I tried to deploy on [Railway](https://railway.app/?referralCode=jj) however it crashes so... I think imma change it to `URL` record if that helps?